hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -50,6 +50,47 @@
       milestone: V2
       tests: ["pattgen_perf"]
     }   
+    {
+      name: cnt_rollover
+      desc: '''
+            Checking ip operation with random counter values
+
+            Stimulus:
+              - Program the pre-divider and size registers to unconstraint random values
+              - Program the clk_cnt, bit_cnt and rep_cnt to values less than but close to
+		predivider and size registers, so that counting would take a reasonable
+		number of clock cycles
+	      - include programming for corner cases
+                repeat programming a random number of times
+              - Start and stop channels quickly
+              - Clear interrupts quickly
+            Checking:
+	      - Include functional cover point that rollover value is reached and counter is reset
+              - Ensure patterns are correctly generated
+              - Ensure interrupts are robust asserted and cleared (e.g. at the high data rate)
+            '''
+      milestone: V2
+      tests: ["pattgen_rand"]
+    }
+    {
+      name: alert
+      desc: '''
+            Checking alert errors
+
+            Stimulus:
+	    To Do: identify exactly what alerts are triggered, and for the V2 
```

Comment:
use TODO and fix the alignment below

Reply:
```suggestion
	    TODO: identify exactly what alerts are triggered, and for the V2 
```

Reply:
I wrote "TODO" and aligned the section below

Reply:
I wrote "TODO" and fixed the alignment

Reply:
I Removed redundant bins and crossed cover points without bins

Reply:
I just aligned and generated a document, and I view both test plan and document in front of me, while the lines in a test plan are aligned, the lines in document are still misaligned. The document was successfully generated (built), committed and pushed to origin